src/core/account/account-init.ts

```diff
@@ -48,8 +49,8 @@ async function createChildLogin(
   const parentKit: LoginKit = {
     serverPath: kit.serverPath,
     server: kit.server,
-    login: { children: [kit.login] },
-    stash: { children: [kit.stash] },
+    login: { children: [kit.login as LoginTree] },
+    stash: { children: [kit.stash as LoginStash] },
```

Comment:
Are these suppose to be size-one vectors?

Reply:
These are arrays with a single element, that we have to type-cast to get rid of the `Partial<>`.